scripts/logextract.py

```diff
@@ -41,7 +41,10 @@ def add_comment(self, comment):
             self.comments.append(comment)
     def write_file(self):
         f = open(self.filename, 'wb')
```

Comment:
can you use `with open(self.filename, 'wb') as f:` instead, for a more modern approach.